Frends.AzureBlobStorage.ReadBlob/README.md

```diff
@@ -0,0 +1,79 @@
+# Frends.AzureBlobStorage.ReadBlob
+FRENDS Task for reading a file from Azure Storage.
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT) 
+[![Build](https://github.com/FrendsPlatform/Frends.AzureBlobStorage/actions/workflows/ReadBlob_build_and_test_on_main.yml/badge.svg)](https://github.com/FrendsPlatform/Frends.AzureBlobStorage/actions)
+![MyGet](https://img.shields.io/myget/frends-tasks/v/Frends.AzureBlobStorage.ReadBlob)
+![Coverage](https://app-github-custom-badges.azurewebsites.net/Badge?key=FrendsPlatform/Frends.AzureBlobStorage/Frends.AzureBlobStorage.ReadBlob|main)
+
+- [Installing](#installing)
+- [Task](#task)
+     - [ReadBlob](#ReadBlob)
+- [Building](#building)
+- [License](#license)
+- [Contributing](#contributing)
```

Comment:
You don't need all of this information in this README.md. This info is enough: https://github.com/FrendsPlatform/Frends.FixedWidthFlatFile/blob/main/Frends.FixedWidthFlatFile.Parse/README.md.